src/embind/embind_gen.js

```diff
@@ -575,7 +583,7 @@ var LibraryEmbind = {
           context,
           isPureVirtual,
           isAsync) {
-    createFunctionDefinition(methodName, argCount, rawArgTypesAddr, context, true, isAsync, (funcDef) => {
+    createFunctionDefinition(methodName, argCount, rawArgTypesAddr, context, true, false, isAsync, (funcDef) => {
```

Comment:
I wonder if its time to make this function take a property bag instead of all those arguments?  (As a followup?)

Reply:
Yeah, it was probably time a few arguments ago.